homework/src/index.js

```diff
@@ -1,47 +1,119 @@
-'use strict';
-
+"use strict";
 {
-  function fetchJSON(url, cb) {
-    const xhr = new XMLHttpRequest();
-    xhr.open('GET', url);
-    xhr.responseType = 'json';
-    xhr.onload = () => {
-      if (xhr.status < 400) {
-        cb(null, xhr.response);
-      } else {
-        cb(new Error(`Network error: ${xhr.status} - ${xhr.statusText}`));
+  function main() {
+    fetchJSON("https://api.github.com/orgs/HackYourFuture/repos", (err, data) => {
+      function rend(changeEvent) {
+        main.innerHTML = "";
+        let sectL = crnap("section", main, { class: "left", id: "maininfo" });
+        const sectR = crnap("section", main, { class: "right", id: "contribInfo" });
+        let sectLinfobox = crnap("div", sectL, { id: "info-box", class: "scy" });
+        sectL = crnap("div", sectLinfobox, {
+          html:
+            null == changeEvent.full_name
+              ? "No name"
+              : `Repo name: <a href="${changeEvent.html_url}">${changeEvent.full_name}</a>`,
+          class: "name"
+        });
+        crnap("div", sectLinfobox, {
+          html:
+            null == changeEvent.description
+              ? "No description"
+              : `Description: ${changeEvent.description}`,
+          class: "description"
+        });
+        crnap("div", sectLinfobox, {
+          html: null == changeEvent.forks ? "No forks" : `Forks: ${changeEvent.forks}`,
+          class: "forks"
+        });
+        crnap("div", sectLinfobox, {
+          html:
+            null == changeEvent.updated_at
+              ? "Update is unknown"
+              : `Updated: ${new Date(changeEvent.updated_at).toLocaleString()}`,
+          class: "updated"
+        });
+        sectLinfobox = crnap("div", sectLinfobox, {
+          html: null == changeEvent.size ? "No size" : `Size: ${changeEvent.size}`,
+          class: "size"
+        });
+        fetchJSON(changeEvent.contributors_url, (err, contributors) => {
+          let fetchToDo = contributors.length;
+          contributors.forEach(contributor => {
+            fetchJSON(contributor.url, (err, user) => {
+              Object.assign(contributor, user);
```

Comment:
This inner function is best defined as separate named function at file level. You should strive for your functions to be relatively short, say not longer than can fit on a computer screen.